app/views/layouts/application.html.erb

```diff
@@ -5,16 +5,17 @@
     <meta name="viewport" content="width=device-width,initial-scale=1">
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
```

Comment:
This line gives my local environment issues. I didn't mean to commit this change, but I don't think it hurts for now.